src/als/ui/prefs_ui.ui

```diff
@@ -209,6 +209,118 @@
+           <string>Minimum alignment stars :</string>
+          </property>
+          <property name="textFormat">
+           <enum>Qt::AutoText</enum>
+          </property>
+          <property name="buddy">
+           <cstring>spn_al_minimum_stars</cstring>
+          </property>
+         </widget>
+        </item>
+        <item row="0" column="1">
+         <widget class="QSpinBox" name="spn_al_minimum_stars">
+          <property name="minimum">
+           <number>1</number>
+          </property>
+          <property name="value">
+           <number>25</number>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </item>
+      <item>
+       <spacer name="horizontalSpacer_3">
+        <property name="orientation">
+         <enum>Qt::Horizontal</enum>
+        </property>
+        <property name="sizeHint" stdset="0">
+         <size>
+          <width>160</width>
+          <height>20</height>
+         </size>
+        </property>
+       </spacer>
+      </item>
+     </layout>
+    </widget>
+   </item>
+   <item>
+    <widget class="QGroupBox" name="groupBox_5">
+     <property name="title">
+      <string>Pre-Processing</string>
+     </property>
+     <layout class="QHBoxLayout" name="horizontalLayout_6">
+      <item>
+       <layout class="QGridLayout" name="gridLayout">
+        <item row="0" column="0">
+          <widget class="QCheckBox" name="chk_use_dark">
+           <property name="text">
+            <string>&amp;Use master dark</string>
+           </property>
+          </widget>
+        </item>
+        <item row="1" column="0">
+         <widget class="QLabel" name="lbl_dark_folder">
+          <property name="text">
+           <string>&amp;Master dark folder :</string>
+          </property>
+          <property name="buddy">
+           <cstring>btn_dark_scan</cstring>
+          </property>
+         </widget>
+        </item>
+        <item row="1" column="1">
+         <widget class="QLineEdit" name="ln_master_dark_path">
+          <property name="minimumSize">
+           <size>
+            <width>350</width>
+            <height>0</height>
+           </size>
+          </property>
+          <property name="readOnly">
+           <bool>true</bool>
+          </property>
+         </widget>
+        </item>
+        <item row="1" column="2">
+         <widget class="QPushButton" name="btn_dark_scan">
+          <property name="text">
+           <string>Change...</string>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </item>
+      <item>
+       <spacer name="horizontalSpacer_4">
+        <property name="orientation">
+         <enum>Qt::Horizontal</enum>
+        </property>
+        <property name="sizeHint" stdset="0">
+         <size>
+          <width>160</width>
+          <height>20</height>
+         </size>
+        </property>
+       </spacer>
+      </item>
+     </layout>
+    </widget>
+   </item>
```

Comment:
globally on the preferences dialog: 

- We miss an accelerator on minimum stars spinbox
- I suggest the following section order. At least, 'misc' should be last :
 pathes, preprocessing, alignment, file saving, web server, misc

Reply:
Ok, I sorted sections.

Reply:
Hi, It seems I don't understand correctly ; could you explain what is an accelerator ?

Reply:
Oh sorry : it is a way of quickly accessing a GUI item via keyboard (Alt+<key>)
Like what you did with the dark section, for example : Alt+U toggles the checkbox

Reply:
Ok, corrections done.